Assets/Scripts/UI/SkillUI.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class SkillUI : MonoBehaviour
+{
+	protected StatUIIconTag _icon = null;
+	protected PlayerStats _playerStats = null;
+
+	public virtual void SetStat( float stat ) { } 
+	public virtual void SetMaxStat( float stat ) { }
```

Comment:
Is there a reason to not make `SetStat()` and `SetMaxStat()` abstract? It seems like that's the more appropriate approach in this case.